src/transaction/components/TransactionProvider.tsx

```diff
@@ -75,6 +84,7 @@ export function TransactionProvider({
     setTransactionId,
     status,
     transactionId,
+    transactionHash: callsStatus?.receipts?.[0]?.transactionHash,
```

Comment:
@abcrane123 let's get an utility for this. something like `getTransactionHash`, and for now have it as `utils` in the transaction module.

Reply:
i combined in the `useCallsStatus` wrapper hook. let me know if you think i should separate them instead.